lib/views/pages/organization/join_organization.dart

```diff
@@ -393,7 +393,7 @@ class _JoinOrganizationState extends State<JoinOrganization> {
         foregroundColor: Colors.white,
         elevation: 5.0,
         onPressed: () {
-          Navigator.of(context).pushReplacement(MaterialPageRoute(
+          Navigator.of(context).push(MaterialPageRoute(
```

Comment:
Keep this one as it is some there is this screen next one on the navigator stack after the Create organization page.